tests/sentry/api/endpoints/test_organization_region.py

```diff
@@ -0,0 +1,134 @@
+from sentry.models.organization import Organization
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.silo import control_silo_test, create_test_regions
+from sentry.types.region import Region, get_region_by_name
+from sentry.utils.security.orgauthtoken_token import generate_token, hash_token
+
+
+@control_silo_test(regions=create_test_regions("us", "de"))
+class OrganizationRegionTest(APITestCase):
+    endpoint = "sentry-api-0-organization-region"
+
+    def setUp(self):
+        super().setUp()
+        self.org_owner = self.create_user()
+        us_region = get_region_by_name("us")
+        self.org = self.create_organization(owner=self.org_owner, region=us_region)
+        self.test_project = self.create_project(organization=self.org, name="test_project")
+
+    def create_internal_integration_for_org(self, org, user, scopes: list[str]):
+        internal_integration = self.create_internal_integration(organization=org, scopes=scopes)
+        integration_token = self.create_internal_integration_token(
+            internal_integration=internal_integration,
+            user=user,
+        )
+
+        return (internal_integration, integration_token)
+
+    def create_auth_token_for_org(self, org: Organization, region: Region, scopes: list[str]):
+        org_auth_token_str = generate_token(org.slug, region.to_url(""))
+        self.create_org_auth_token(
+            organization_id=org.id,
+            scope_list=scopes,
+            name="test_token",
+            token_hashed=hash_token(org_auth_token_str),
+            date_last_used=None,
+        )
+
+        return org_auth_token_str
+
+    def test_org_member_has_access(self):
+        self.login_as(self.org_owner)
+        response = self.get_response(self.org.slug)
+        assert response.status_code == 200
+
+    def test_non_org_member_has_no_access(self):
+        non_member_user = self.create_user()
+        self.login_as(non_member_user)
```

Comment:
Should we have tests for user tokens as well? The `login_as` method simulates cookie authentication.

Reply:
Ahh yep, I'll go ahead and add those as well.